### PR TITLE
Adds trait-dependent room moodlet check

### PR DESCRIPTION
**Pull request by RaveRadbury**
## About The Pull Request
This adds a check for traits to be included with room moodlets. I have added one for the Spiritual quirk, gained by being in the chapel areas. It is +5, equal to the bar moodlet.

I plan on doing more with this, but for the sake of atomization I've only provided one implementation of the check.

## Why It's Good For The Game
This adds more texture to quirks and influences where players might choose to spend their time.

## Changelog
:cl:
add: Spiritual crew get a mood boost from being in the Chapel
/:cl:

### CODE REVIEW

File: code/game/area/space_station_13_areas.dm [hunk 1]
```diff
@@ -534,6 +534,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 
 /area/chapel
 	icon_state = "chapel"
+	mood_bonus = 5
+	mood_message = "<span class='nicegreen'>Being in the chapel brings me peace.\n</span>"
```
Comment (coiax) [inline code/game/area/space_station_13_areas.dm:538]:
```suggestion
	mood_message = "<span class='nicegreen'>Being in the chapel brings me peace.</span>\n"
```

Comment (RaveRadbury) [reply code/game/area/space_station_13_areas.dm:538]: The bar and psych office both also put the `\n` before the `</span>` Should they be changed as well?

Comment (coiax) [reply code/game/area/space_station_13_areas.dm:538]: Mmm, good question. All the mood messages that I've seen from regular moodlets, all have the `\n` outside the span. So yeah, change those too.

Comment (RaveRadbury) [reply code/game/area/space_station_13_areas.dm:538]: Should I do that here, or in a separate PR?

Comment (coiax) [reply code/game/area/space_station_13_areas.dm]: Do it in a seperate PR, this one's ready to go.

**Pull request merged**